sdcm/sct_provision/aws/cluster.py

```diff
@@ -288,6 +288,7 @@ def _user_data(self) -> str:
         return AWSInstanceUserDataBuilder(
             params=self.params,
             syslog_host_port=self._test_config.get_logging_service_host_port(),
+            aws_additional_interface=True,
```

Comment:
The loader should be able to connect to the DB nodes and run stress command.
If the test runs with independent multi networks, stress may fails because the second interface
is not configured on the loader and cannot reach second interface of DB nodes.

So adding of `aws_additional_interface=True` allow to configure second network on the loader as well as on DB nodes.

Reply:
yes, I meant why setting it as default

Reply:
good catch. Fixed